docs/usage.rst

```diff
@@ -34,9 +34,27 @@ Non-blocking lock::
     else:
         print("Someone else has the lock.")
 
-Controlled execution lock::
+Releasing previously acquired lock::
+
+    conn = StrictRedis()
+    lock = redis_lock.Lock(conn, "name-of-the-lock")
+    lock.acquire()
+    print("Got the lock. Doing some work ...")
+    time.sleep(5)
+    lock.release()
+
+The above example could be rewritten using context manager::
 
     conn = StrictRedis()
     with redis_lock.Lock(conn, "name-of-the-lock"):
         print("Got the lock. Doing some work ...")
         time.sleep(5)
+
+If lock was acquired in other context (other process, remote worker server,
+etc), it's still possible to release it with parameter `force` equal
+to ``True``::
```

Comment:
This is incorrect. In order to release locks held from other places you need to use `reset()`. The force option is there for sloppy code that don't always acquire it (or didn't check if it was acquired properly).

Example code:

```python

lock = Lock(conn, "foo")
try:
  if lock.acquire(block=False):
    print("Got the lock. Do crazy dance")
  else:
    print("Didn't get the lock. Do normal dance")
finally:
  lock.release(force=True)
```